scripts/sync_gokb_packages.groovy

```diff
@@ -59,23 +59,21 @@ while ( moredata ) {
           }
 
           // Title.
-          newtipp.title = addCoreItems ( xmltipp.title )
-
-          String type = "${cleanText(xmltipp.title.type?.text())}"
-
-          if (!type || type == 'JournalInstance' || type == 'Serial') {
-            newtipp.title.type = 'Serial'
-          }
-          else if (type == 'DatabaseInstance') {
-            newtipp.title.type = 'Database'
+          if (xmltipp.title){
+            newtipp.title = addCoreItems ( xmltipp.title )
+            String type = "${cleanText(xmltipp.title.type?.text())}"
+            if (!type || type == 'JournalInstance' || type == 'Serial') {
+                newtipp.title.type = 'Serial'
+            }
+            else if (type == 'DatabaseInstance') {
+                newtipp.title.type = 'Database'
+            }
+            else if (type == 'BookInstance'|| type == 'Monograph') {
+                newtipp.title.type = 'Book'
+            }
+            newtipp.platform = directAddFields (xmltipp.platform, ['primaryUrl'], addCoreItems ( xmltipp.platform ))
```

Comment:
The TIPP platform is not depending on title information.

Reply:
Moved line 74 out of the title check.